go/src/koding/klient/machine/machinegroup/clients/clients_test.go

```diff
@@ -0,0 +1,71 @@
+package clients
```

Comment:
we don't need to access internal implementation in any of `machine` package-related tests.